modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/service/test/ObjectRelationshipLocalServiceTest.java

```diff
@@ -514,6 +514,66 @@ public void testBindPublishedObjectDefinitions() throws Exception {
 					_treeFactory.createObjectDefinitionTree(
 						objectDefinition1.getObjectDefinitionId()),
 					_objectDefinitionLocalService));
+
+		objectDefinitionA = _addAndPublishCustomObjectDefinition();
+		objectDefinitionAA = _addAndPublishCustomObjectDefinition();
+
+		_createObjectRelationshipEdge(objectDefinitionA, objectDefinitionAA);
+
+		ObjectDefinition objectDefinitionAB =
+			_addAndPublishCustomObjectDefinition();
+
+		_createObjectRelationshipEdge(objectDefinitionA, objectDefinitionAB);
+
+		ObjectDefinition objectDefinitionAAA =
+			_addAndPublishCustomObjectDefinition();
+		ObjectDefinition objectDefinitionAAAA =
+			_addAndPublishCustomObjectDefinition();
+
+		_createObjectRelationshipEdge(
+			objectDefinitionAAA, objectDefinitionAAAA);
+
+		ObjectDefinition objectDefinitionAAAAA =
+			_addAndPublishCustomObjectDefinition();
+
+		_createObjectRelationshipEdge(
+			objectDefinitionAAAA, objectDefinitionAAAAA);
+
+		_createObjectRelationshipEdge(objectDefinitionAA, objectDefinitionAAA);
```

Comment:
Does it make a difference to run
`_createObjectRelationshipEdge(objectDefinitionAAAA, objectDefinitionAAAAA);
`before this
`_createObjectRelationshipEdge(objectDefinitionAA, objectDefinitionAAA); `
to your test scenario?

If not, can we sort and remove the empty line?
```
_createObjectRelationshipEdge(objectDefinitionAA, objectDefinitionAAA);
_createObjectRelationshipEdge(objectDefinitionAAAA, objectDefinitionAAAAA);
```